spec-main/fixtures/api/nativeimage-serialization.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title></title>
+  <body>
+    <script>
+      const { ipcRenderer, remote, nativeImage } = require('electron')
+      const template = [
+        {
+          label: 'hello',
+          icon: nativeImage.createEmpty(),
+        }
+      ]
+
+      try {
+        const menu = remote.Menu.buildFromTemplate(template)
```

Comment:
This is a bit of a roundabout way to check the serialization. Perhaps just `remote.getGlobal('someFunction')(nativeImage.createEmpty())`?

Reply:
Also, to avoid having to make an actual global, you can intercept the global filtering stuff to provide a test-specific response. see the other tests in `api-remote-spec` for examples.

Reply:
@nornagon i'm not sure i can use that approach - i believe we specifically only handle simpler objects over [`valueToMeta`](https://github.com/electron/electron/blob/f0737174efe3664f7e6e830b339517680e352ba5/lib/browser/remote/server.ts#L120)? If i set a native image on the return value for that kind of filtering it'll just show up as empty object 🤔